tools/bin/go_core_ccip_deployment_tests

```diff
@@ -6,22 +6,23 @@ SCRIPT_PATH=`dirname "$0"`; SCRIPT_PATH=`eval "cd \"$SCRIPT_PATH\" && pwd"`
 OUTPUT_FILE="../output.txt"
 USE_TEE="${USE_TEE:-true}"
 
-# To allow reuse in CI from other repositories
-TOOLS_PATH=${TOOLS_PATH:-"./tools"}
-
+cd ./deployment || exit
+go mod download
 echo "Failed tests and panics: ---------------------"
 echo ""
-use_tee() {
-  if [ "$USE_TEE" = "true" ]; then
-    tee "$@"
+if [ "$GITHUB_EVENT_NAME" == "schedule" ]; then
```

Comment:
should we do double brackets? `[[ ]]`

Reply:
282d22f2e21aab12762fd8ada27b03af7a698df8